modules/about-etcd-encryption.adoc

```diff
@@ -23,5 +23,5 @@ When you enable etcd encryption, encryption keys are created. These keys are rot
 ====
 Keep in mind that etcd encryption only encrypts values, not keys. This means that resource types, namespaces, and object names are unencrypted.
 
-During etcd backup, if etcd encryption is enabled then static_kuberesources_<datetimestamp>.tar.gz: file also contains the encryption keys for the etcd snapshot and it is recommended to store this file separately from the etcd snapshot for security reasons. However, this file is required to restore a previous state of etcd from respective etcd snapshot.
+If etcd encryption is enabled during a backup, the `__static_kuberesources_<datetimestamp>.tar.gz: file__` contains the encryption keys for the etcd snapshot. You should store this file separately from the etcd snapshot for security reasons. However, this file is required to restore a previous state of etcd from the respective etcd snapshot.
```

Comment:
@asme30 To keep the text aligned with IBM and Red Hat style guidance, please update the second sentence from `You should store this file separately...` to `Store this file separately...`. After that change has been made, I can help you move this forward to be reviewed and merged by the OpenShift docs team. Thanks!